From 63687ad1247ac2313b3553c70ccd372bea3472da Mon Sep 17 00:00:00 2001
From: Mirco Tischler <mircotischler@uni-wuppertal.de>
Date: Tue, 23 May 2017 23:11:45 +0200
Subject: [PATCH] Remove 'lib' prefix when looking for libraries

ld will prepend the lib prefix itself and will actually look for
liblibgpg-error and liblibm if called with -llibgpg-error and -llibm. Let's
get rid of the prefix.
---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index fae641b..1751256 100644
--- a/meson.build
+++ b/meson.build
@@ -122,11 +122,11 @@ appstream_glib = dependency('appstream-glib', version : '>= 0.5.10')
 gusb = dependency('gusb', version : '>= 0.2.9')
 sqlite = dependency('sqlite3')
 libarchive = dependency('libarchive')
-valgrind = dependency('valgrindXXX', required: false)
+valgrind = dependency('valgrind', required: false)
 soup = dependency('libsoup-2.4', version : '>= 2.51.92')
 gpgme = cc.find_library('gpgme')
-gpgerror = cc.find_library('libgpg-error')
-libm = cc.find_library('libm', required: false)
+gpgerror = cc.find_library('gpg-error')
+libm = cc.find_library('m', required: false)
 systemd = dependency('systemd')
 udev = dependency('udev')
 
-- 
2.13.0

